packages/plugin-commands-installation/src/recursive.ts

```diff
@@ -420,6 +421,25 @@ async function readLocalConfig (prefix: string) {
   }
 }
 
+function calculateRepositoryRoot (
+  workspaceDir: string,
+  projectDirs: string[]
+) {
+  // assume repo root is workspace dir
+  let relativeRepoRoot = '.'
+  for (const rootDir of projectDirs) {
+    const relativePartRegExp = new RegExp(`^(\\.\\.\\${path.sep})+`)
+    const relativePartMatch = relativePartRegExp.exec(path.relative(workspaceDir, rootDir))
```

Comment:
Is a regex really needed here? Seems like a simle **.startsWith(\`..${path.sep}\`)** would be enough.